frontend/.gitignore

```diff
@@ -0,0 +1,43 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
It seems weird that there are two `.gitignore` files. This one seems more full, but I would expect it at the root where the other is. If possible, we should consolidate into one of them.